code/lib/core-server/src/utils/new-story-templates/typescript.ts

```diff
@@ -0,0 +1,33 @@
+import dedent from 'ts-dedent';
+
+export function getTypeScriptTemplateForNewStoryFile(data: {
+  /** The components file name without the extension */
+  basename: string;
+  componentExportName: string;
+  default: boolean;
+  /** The framework package name, e.g. @storybook/nextjs */
+  frameworkPackageName: string;
+  /** The exported name of the default story */
+  exportedStoryName: string;
+}) {
+  const importName = data.default ? 'Component' : data.componentExportName;
+  const importStatement = data.default
+    ? `import ${importName} from './${data.basename}'`
+    : `import { ${importName} } from './${data.basename}'`;
```

Comment:
Should we be considering their prettier/eslint/editorconfig settings regarding quotes and semicolons? Perhaps infer from an existing CSF file?